Modules/IO/TransformFactory/src/itkTransformFactoryBaseRegister.hxx

```diff
@@ -178,6 +178,8 @@ TransformFactoryBase::RegisterTransformFactory()
   TransformFactory<TimeVaryingVelocityFieldTransform<TParameterType, 3>>::RegisterTransform();
   TransformFactory<VelocityFieldTransform<TParameterType, 2>>::RegisterTransform();
   TransformFactory<VelocityFieldTransform<TParameterType, 3>>::RegisterTransform();
+
+  TransformFactory<MatrixOffsetTransformBase<TParameterType, 3, 3>>::RegisterTransform();
```

Comment:
This is a base class and is not meant to be instantiated directly, so should it still be added here?